person.rb

```diff
@@ -0,0 +1,42 @@
+require_relative 'nameable'
+require_relative 'decorator'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+
+class Person < Nameable
```

Comment:
- [ ] Hi @Zuwairanajma, let's again kindly make sure the relationship between person and rentals is correctly configure. A person has many rentals: Let's make sure Getters and Setters for @rentals  is present. You rightly added method for adding rentals but didn't initialized the @rentals instance variable to an empty array. Please kindly fix these minor changes to better prepare your project for approval :heavy_check_mark: